optuna/study/study.py

```diff
@@ -945,6 +949,47 @@ def objective(trial):
         for trial in trials:
             self.add_trial(trial)
 
+    def set_metric_names(self, metric_names: List[str]) -> None:
+        """Set metric names.
+
+        Each dimension of the objective function is named for each element of the
+        `metric_names` argument.
```

Comment:
nit: Is it possible to explain more clearly how this method is used?

e.g. This method names each dimension of the returned values of the objective function. It is particularly useful in multi-objective optimization. The metric names are mainly referenced by the visualization functions.